flow/envs/green_wave_env.py

```diff
@@ -290,9 +290,16 @@ def _convert_edge(self, edges):
 
         The numbers are assigned along the lowest column, then the lowest row,
         then the second lowest column, etc. Left goes before right, top goes
-        before bot.
+        before bottom.
+
+        The values are zero indexed.
+
+        Example for a 3x3 grid network:
```

Comment:
This is 4x4, no?

Reply:
I'll actually remove that comment, since I think it's wrong. I was trying to understand that method. I don't think I understand it, but I ended up not needing it. (But the reason 3x3 is not crazy is because the method is return edge ids and not node ids.)